llvm/CMakeLists.txt

```diff
@@ -905,18 +905,6 @@ if(NOT LLVM_ENABLE_NEW_PASS_MANAGER)
                       " no longer supported.")
 endif()
 
-set(DPCPP_ENABLE_OPAQUE_POINTERS TRUE CACHE BOOL
-   "Enable opaque pointers by default.")
-if (DPCPP_ENABLE_OPAQUE_POINTERS)
-  add_definitions("-DENABLE_OPAQUE_POINTERS=1")
-  add_definitions("-DINTEL_SYCL_OPAQUEPOINTER_READY=1")
```

Comment:
Do we need to remove this line too https://github.com/intel/llvm/blob/7a9c466c19230b3b82c7ce58366cc4815aeb6c6d/llvm/cmake/modules/TableGen.cmake#L7

Reply:
Good catch. Removed.